bluepyemodel/evaluation/evaluator.py

```diff
@@ -126,19 +126,23 @@ def define_recording(recording_conf, use_fixed_dt_recordings=False):
     return rec_class(name=recording_conf["name"], location=location, variable=variable)
 
 
-def define_protocol(protocol_configuration, stochasticity=False, use_fixed_dt_recordings=False):
+def define_protocol(protocol_configuration, stochasticity=False, dt=None, use_fixed_dt_recordings=False):
     """Create a protocol.
 
     Args:
         protocol_configuration (ProtocolConfiguration): configuration of the protocol
         stochasticity (bool): Should the stochastic channels be stochastic or
             deterministic
-        use_fixed_dt_recordings (bool): whether to record at a fixed dt of 0.1 ms.
-
+        dt (float): NEURON dt for fixed time step simulation. If None, variable 
+            dt will be used.
+        use_fixed_dt_recordings (bool): used for legacy currentscape 
+            to record at a fixed dt of 0.1 ms. 
+            However, Simulations will be run based on pipeline_settings.neuron_dt.
+        
     Returns:
         Protocol
     """
-    cvode_active = False if EModelPipelineSettings.neuron_dt is None else True
+    cvode_active = False if dt is None else True
```

Comment:
`True if dt is None else False`

Reply:
made this `cvode_active = not dt` instead.